### DISCUSSION TITLE
[BP#67] Suppression de la pratique "Remplacer les $i++ par ++$i"

### DISCUSSION THREAD

**Discussion by fzaninotto** [cnumr/best-practices#99] [Bonnes pratiques] [answered]
La différence entre les deux syntaxes est négligeable en PHP:

![image](https://user-images.githubusercontent.com/99944/138921646-9e590dcf-12b2-4a20-8bb9-c6915d6e9607.png)

https://3v4l.org/0mLSR

La différence est inexistante dans les autres langages.

![image](https://user-images.githubusercontent.com/99944/138921326-67ac0cbf-3798-42c5-a713-802ca4d43293.png)

source: https://stackoverflow.com/a/24887/1333479

Ref. https://github.com/cnumr/best-practices/blob/main/chapters/BP_067_fr.md

**Comment by tbroyer** [comment, marked as answer]
Doublon de https://github.com/cnumr/best-practices/discussions/19
